### PR TITLE
Implement callback for signal LED state change

### LINKED ISSUE #91: The "Signal" LED does not work from Linux.
Works fine for rt-kernel but needs to be implemented in src/osal/linux/osal.c

Found with Automated RT Tester  case DCP_Signal

### PR DESCRIPTION
**Pull request by pyhys**
Remove LED and button related functionality from OSAL.

Closes #91

### CODE REVIEW

File: src/osal/rt-kernel/osal.c [hunk 1]
```diff
@@ -13,24 +13,19 @@
  * full license information.
  ********************************************************************/
 
-#include "osal.h"
-#include "options.h"
 #include "log.h"
+#include "options.h"
+#include "osal.h"
```
Comment (hefloryd) [inline src/osal/rt-kernel/osal.c:18]: Please keep the module header first in the list of includes (in all files).

Comment (pyhys) [reply src/osal/rt-kernel/osal.c]: Yes, sorry.
Done.
Also cmake change to copy the new scripts.

File: cmake/Linux.cmake [hunk 2]
```diff
@@ -68,6 +68,18 @@ file(COPY
   ${PROFINET_BINARY_DIR}/
   )
 
+file(COPY
```
Comment (hefloryd) [inline cmake/Linux.cmake:71]: You can copy multiple files with one file(COPY), as long as the destionation is the same. Also, I think we should use install() here. The main difference between install() and file(COPY) is that the copy takes place at configuration time (when running cmake). If you wanted to install into /usr/local or similar you'd probably need to be root or similar just to run cmake. Whereas with install() you'd only need root for make install.

Comment (pyhys) [reply cmake/Linux.cmake]: Done. Also adjusted the tests now that #63 have been merged.

Comment (pyhys) [reply cmake/Linux.cmake]: Changed back to file(COPY) after discussions.

**Pull request merged**